src/components/ResultsTable/ResultsTable.styles.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+
+const ResultsTableWrapper = styled.div`
```

Comment:
The `ResultsTable*` prefix can be removed IMO since it's clear from the filename